base_partner_sequence/__openerp__.py

```diff
@@ -1,54 +1,22 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>).
-#    Copyright (C) 2013 initOS GmbH & Co. KG (<http://www.initos.com>).
-#    Author Thomas Rehn <thomas.rehn at initos.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# Copyright 2004-2009 Tiny SPRL (<http://tiny.be>).
+# Copyright 2013 initOS GmbH & Co. KG (<http://www.initos.com>).
+# Copyright 2016 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Add a sequence on customers' code",
-    "version": "8.0.1.1.0",
-    "author": "Tiny/initOS GmbH & Co. KG,Odoo Community Association (OCA)",
+    "version": "9.0.0.1.0",
+    "author": "Tiny/initOS GmbH & Co. KG,"
+              "ACSONE SA/NV,"
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
     "category": "Generic Modules/Base",
     "website": "http://www.initos.com",
-    "depends": ["base"],
```

Comment:
I know ordinarily there is an implicit dependency on base, but in this case, because the module is directly referencing base xml ids I think it would be better to explicitly state.  I've only done a static review and not tested, so don't know whether this would ever actually cause a problem on a fresh database.